radix-engine-interface/src/blueprints/pool/two_resource_pool/invocations.rs

```diff
@@ -0,0 +1,90 @@
+use crate::blueprints::macros::*;
+use crate::blueprints::resource::*;
+use radix_engine_common::data::manifest::model::*;
+use radix_engine_common::math::*;
+use radix_engine_common::prelude::*;
+use radix_engine_common::*;
+
+define_invocation! {
```

Comment:
Thoughts on this? Thinking of expanding this further as it seems like we need it.